frontend/src/features/admin-form/settings/components/AuthSettingsSection/AuthSettingsSection.tsx

```diff
@@ -131,14 +125,30 @@ export const AuthSettingsSection = ({
 
   return (
     <Box>
+      <Text
+        textStyle="body-1"
```

Comment:
```suggestion
        textStyle="subhead-1"
```
Should be `subhead-1` instead based on figma!

Reply:
I think you might need to refactor a bit cos the words and link have diff typography